src/Symfony/Bundle/SecurityBundle/Resources/config/security_listeners.xml

```diff
@@ -258,6 +258,15 @@
             <argument type="service" id="event_dispatcher" on-invalid="null"/>
         </service>
 
+        <service id="security.authentication.sessionexpiration_listener" class="Symfony\Component\Security\Http\Firewall\SessionExpirationListener" public="false" abstract="true">
```

Comment:
I would call it ``session_expiration_listener`` (separating all words with underscores, not only half of them)